quelpa.el

```diff
@@ -156,6 +162,8 @@ If nil the update is disabled and the repo is only updated on
 (defvar quelpa-recipe '(quelpa :url "https://framagit.org/steckerhalter/quelpa.git" :fetcher git)
   "The recipe for quelpa.")
 
+(defvar quelpa--use-local-p nil "Use local checkout instead as package source.")
```

Comment:
I'm not sure I understand exactly what this variable controls.  When it is non-nil, does that mean that Quelpa will build the package from whatever state the working tree is in?

If so, maybe we should rename it to be more descriptive.  As it is, "local" makes me think of "local vs. remote", which isn't necessarily the point, because the current state of the working tree may be a commit pulled from a remote repo.

What do you think?

Reply:
I change the name to `quelpa--use-current-state-p` and update the doc